front/src/components/UI/PhotoButton/PhotoButton.jsx

```diff
@@ -17,7 +17,7 @@ import {
   saveCroppedPhoto,
 } from '../../../store/slices/form/formSlice';
 
-const PhotoButton = () => {
+const PhotoButton = (error) => {
```

Comment:
```suggestion
const PhotoButton = ({ error }) => {
```

это исправит ошибку с тем, что кнопка красная при открытии страницы